packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -18,8 +20,10 @@ import '../config_specific/logger/logger.dart';
 import '../globals.dart';
 import '../service_manager.dart';
 import '../table.dart';
+import '../ui/analytics_constants.dart';
 import '../ui/utils.dart';
 import '../utils.dart';
+
```

Comment:
remove new line here

Reply:
Done, although I do like the blank link separating relative from same directory.